src/Resources/config/config.yaml

```diff
@@ -70,6 +70,11 @@ sylius_ui:
                 pay_pal_payment_method:
                     template: '@SyliusPayPalPlugin/javascripts.html.twig'
                     priority: -100
+        sylius.admin.order.show.payment_content:
+            blocks:
+                pay_pal_refunded:
```

Comment:
Just a suggestion,
```patch
--- pay_pal_refunded
+++ pay_pal_refunded_message
```
or plain `pay_pal_message` -- `complete_transition` or `refund_transition` will show up pr be hidden depending on underling state as well.

Reply:
let's go with `pay_pal_refunded_message`, more precise in its intent